.storybook/test-data.js

```diff
@@ -571,7 +571,26 @@ const state = {
         balance: '0x15f6f0b9d4f8d000',
       },
     },
+    accountsByChainId: {
+      '0x5': {
+        '0x64a845a5b02460acf8a3d84503b0d68d028b4bb4': {
+          address: '0x64a845a5b02460acf8a3d84503b0d68d028b4bb4',
+          balance: '0x176e5b6f173ebe66',
+        },
+        '0xb19ac54efa18cc3a14a5b821bfec73d284bf0c5e': {
+          address: '0xb19ac54efa18cc3a14a5b821bfec73d284bf0c5e',
+          balance: '0x2d3142f5000',
+        },
+        '0x9d0ba4ddac06032527b140912ec808ab9451b788': {
+          address: '0x9d0ba4ddac06032527b140912ec808ab9451b788',
+          balance: '0x15f6f0b9d4f8d000',
```

Comment:
🤔 this is the first thing I'm seeing in this PR but it strikes me as strange, will the "address" value ever diverge from the key for the outer object? And if not do we need the "address" property at all?

Reply:
I don't believe the address values will ever be mismatched. I'm not sure that we need the address property at all. Probably not something we need to address as part of this PR